classes/Product.php

```diff
@@ -7090,16 +7090,13 @@ public function getWsProductOptionValues()
      */
     public function getWsPositionInCategory()
     {
-        $result = Db::getInstance()->executeS(
+        $position = (int) Db::getInstance()->getValue(
```

Comment:
Hi ! I checked with @Amoifr and @saulaski and we believe `getValue()` never returns `null`

Moreover you cast the result to int, so it will never be null, so the coalesce operator is not relevant, wdyt ?

Ref https://github.com/PrestaShop/PrestaShop/blob/develop/classes/db/Db.php#L692

Reply:
Moreover, since the coalesce operator will never be fed `null`, then this new implementation will always return a numeric, never a bool, now?